daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -51,29 +53,35 @@ func Init(home string, options []string) (graphdriver.Driver, error) {
 	return graphdriver.NaiveDiffDriver(driver), nil
 }
 
+// Driver contains information about the filesystem mounted.
 type Driver struct {
+	//root of the file system
 	home string
 }
 
+// String prints the name of the driver (btrfs).
 func (d *Driver) String() string {
 	return "btrfs"
 }
 
+// Status returns current driver information.
```

Comment:
@stevvooe There is no plural here. It is as simple as returning a status. Can you elaborate?